lib/LaTeXML/Common/Font.pm

```diff
@@ -266,6 +276,7 @@ sub stringify {
   no warnings 'recursion';
   my ($self) = @_;
   my ($fam, $ser, $shp, $siz, $col, $bkg, $opa, $enc, $lang, $mstyle, $flags) = @$self;
+  # !!!!!
```

Comment:
What could be some words to describe that exclamation?

Reply:
that got added? Hmm...